include/micm/util/sparse_matrix_vector_ordering.hpp

```diff
@@ -4,9 +4,11 @@
  */
 #pragma once
 
+#include <functional>
 #include <micm/util/matrix_error.hpp>
 #include <micm/util/vector_matrix.hpp>
 
+#include "sparse_matrix.hpp"
```

Comment:
For consistency in including project header files, could you change this?
```suggestion
#include <micm/util/sparse_matrix.hpp>
```

Reply:
Updated!